src/ADSB/ADSBVehicleManager.cc

```diff
@@ -18,6 +18,7 @@
 ADSBVehicleManager::ADSBVehicleManager(QGCApplication* app, QGCToolbox* toolbox)
     : QGCTool(app, toolbox)
 {
+    qmlRegisterUncreatableType<ADSBVehicle>("QGroundControl.ADSBVehicle",1, 0, "ADSBVehicle","Reference only");
```

Comment:
I didn't notice this until now, I have no idea what it is. I think it came from Markus' commit